### ISSUE TITLE
Error E0107 - False positive while using tauri

### ISSUE THREAD

**Issue by RobDeFlop** [intellij-rust/intellij-rust#9128]
## Environment

* **IntelliJ Rust plugin version:** 0.4.175.4772-222
* **Rust toolchain version:** 1.62.1 (e092d0b6b 2022-07-16) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2022.2 (CL-222.3345.126)
* **Operating system:** Windows 11 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description
When you use Tauri and create a command to receive an event from the "frontend" and want to access the window as a param, it throws an E0107 Error for the `Window` data type that is a false positive. Reference is the [official Tauri documentation](https://tauri.app/v1/guides/features/command#accessing-the-window-in-commands)

![grafik](https://user-images.githubusercontent.com/25180999/181599121-7ffec3fc-f52d-4f67-9e37-207768fea8f3.png)

## Steps to reproduce
Set up a tauri app and add an event to receive the invoked event from the "frontend". 

```rs
#[tauri::command(async)]
async fn app_ready(window: Window) {
    println!("Received Event");
}
```

**Comment by sekwah41** [comment]
Just found this too now. Glad its reported.

**Comment by sekwah41** [comment]
I think technically the right way to state it is 
```rust
#[tauri::command]
fn set_ui_theme<R: Runtime>(is_dark_mode: bool, _window: tauri::Window<R>) {
    println!("set_ui_theme! {}", is_dark_mode);
}
````

So this plugin actually is probably correct to warn about it, but its doing red warnings against legit rust code.

**Comment by Undin** [comment]
The plugin shows error annotation in this case because it doesn't expand attribute procedural macros by default for now.
As a result, it shows false positive error here.
But you can enable expansion of attributes macros with `org.rust.macros.proc.attr` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features) and it should solve your issue.
Don't forget to [reload the project model](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-cargo-tool-window.html#cargo-load) via `Refresh Cargo Projects` action in Cargo tool window after it.
See #6908 for more details
